extra_time_application/models/project_project_inherit.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ProjectProjectInherit(models.Model):
+    _inherit = 'project.project'
+
+    is_modified = fields.Boolean()
```

Comment:
What is this for?
This column is never assigned and has no title.
What is the usage?
cc @lonelysun